tests/test_component_media.py

```diff
@@ -1047,6 +1046,13 @@ class Media:
         assertInHTML('<script src="child.js"></script>', rendered)
         assertInHTML('<script src="parent.js"></script>', rendered)
 
+        assert str(ChildComponent.media) == (
```

Comment:
Unrelated to this PR, but added tests to check that coercing to string the `Component.media` attribute behaves same as in Django, where it prints out all JS / CSS as `<link>` / `<script>` tags.